.travis.yml

```diff
@@ -44,17 +43,17 @@ install:
       export CXX=g++-4.8;
     fi
 before_script:
-  - export CXXFLAGS="-Werror"
   - $CXX --version
   - xml2-config --version
   - geos-config --version
   - proj | head -n1
   - lua -v
 script:
-  - ./autogen.sh
-  - ./configure
-  - make -j2 check TESTS='' CXXFLAGS="$BUILDCXXFLAGS"
+  - mkdir build && cd build
+  - cmake .. -DBUILD_TESTS=ON -DCMAKE_BUILD_TYPE=Debug $CMAKEOPTS
```

Comment:
This can be needed if we test different options on Travis, but currently it can be removed.